source/features/merged-pr-first-published-tag.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import TagIcon from 'octicon/tag.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../libs/features';
+import fetchDom from '../libs/fetch-dom';
+import {getRepoURL} from '../libs/utils';
+import observeElement from '../libs/simplified-element-observer';
+
+const getFirstTag = cache.function(async (commit: string): Promise<[string, string] | undefined> => {
+	const firstTag = await fetchDom<HTMLAnchorElement>(`/${getRepoURL()}/branch_commits/${commit}`, 'ul.branches-tag-list li:last-child a');
+	if (!firstTag) {
+		return;
+	}
+
+	return [firstTag.href, firstTag.textContent!];
+}, {
+	cacheKey: ([commit]) => `first-tag:${getRepoURL()}:${commit}`
+});
+
+async function addTag(discussionHeader: HTMLElement): Promise<void | false> {
+	const mergeCommit = select('div.TimelineItem-body > a[href*="commit"] > code.link-gray-dark')!.textContent!;
+	const [tagUrl, tagName] = await getFirstTag(mergeCommit) ?? [];
+
+	if (!tagUrl) {
+		return;
+	}
+
+	discussionHeader.parentElement!.append(
+		<span>
+			• <TagIcon width={14} className="mx-1 text-gray-light" style={{paddingTop: '1px'}}/>
```

Comment:
```suggestion
			• <TagIcon width={14} className="mx-1 text-gray-light v-align-middle"/>
```

Also I don't see the need to resize it
```suggestion
			• <TagIcon className="mx-1 text-gray-light v-align-middle"/>
```